src/dcm-taskdefs.ant.xml

```diff
@@ -1786,7 +1786,7 @@
         <domain>@{domain}</domain>
         <local>@{inputfile}</local>
         <deployment-policy>@{dpo}</deployment-policy>
-        <deployment-policy-variables>@{dpov}</deployment-policy-variables>
+	<deployment-policy-variables>@{dpov}</deployment-policy-variables>
```

Comment:
spacing

Reply:
I don't know, from where this spacing is coming. When i edit this file here it, does not show this space.

Reply:
It's your IDE. I think you used a tab instead of a space or spaces instead of a tab.